tools/sdk-ai-bots/.pipelines/build-document-embeddings.yml

```diff
@@ -35,12 +35,13 @@ stages:
     - template: setup-pipeline.yml
     - checkout: git://internal/_git/azure-sdk-docs-eng.ms
       displayName: 'Checkout azure-sdk-docs-eng.ms repository'
-    - task: Powershell@2
+    - task: AzurePowerShell@5
       inputs:
-        filePath: $(Build.SourcesDirectory)/azure-sdk-tools/tools/sdk-ai-bots/Scripts/Build-EngHubDocEmbeddings.ps1
-        arguments: >
-          -IncrementalEmbedding "${{ parameters.incrementalEmbedding }}"
-        pwsh: true
+        azureSubscription: 'Azure SDK Playground'
```

Comment:
Can we make this as a parameter instead of hard coded value?

Reply:
Updated.